pci-dss/add-annotations.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: add-annotations
+  annotations:
+    policies.kyverno.io/title: Add annotations
+    policies.kyverno.io/category: Sample
```

Comment:
We do not want a `Sample` category. It can be `Best Practices`, or perhaps something else?